apps/meeting/src/containers/Chat/ChatInput.tsx

```diff
@@ -0,0 +1,34 @@
+import { Input } from 'amazon-chime-sdk-component-library-react';
+import React, { ChangeEvent, useState } from 'react';
+import { useDataMessages } from '../../providers/DataMessagesProvider';
+import { StyledChatInputContainer } from './Styled';
+
+export default function ChatInput() {
+  const [message, setMessage] = useState('');
+  const { sendMessage } = useDataMessages();
```

Comment:
Nope as of now this is just to unblock us on the communications in standups. But still since React meeting demo is good on UI, I tried to make it as ready as I can. (There are no performance improvements or grouping by name as such but I plan to add once I get more time)